### PR TITLE
Add ShowCommand and ShowCommandParser as well as their tests

### LINKED ISSUE #63: Add ShowCommand and ShowCommandParser

### PR DESCRIPTION
**Pull request by anthony6401**
Resolve #63

### CODE REVIEW

File: src/test/java/seedu/address/model/AddressBookTest.java [hunk 1]
```diff
@@ -44,8 +44,8 @@ public void resetData_withValidReadOnlyAddressBook_replacesData() {
     @Test
     public void resetData_withDuplicateTags_throwsDuplicateTagException() {
         // Two persons with the same identity fields
-        Tag editedAlice = new TagBuilder(ALICE).build();
-        List<Tag> newPersons = Arrays.asList(ALICE, editedAlice);
+        Tag editedAlice = new TagBuilder(CS2103).build();
```
Comment (luo-git) [inline src/test/java/seedu/address/model/AddressBookTest.java]: Maybe the name of the variables here could be changed?

**Review by luo-git — CHANGES_REQUESTED** [review]
Good work. Minor changes proposed.

File: src/test/java/seedu/address/model/AddressBookTest.java (hunk 1 above)
Comment (anthony6401) [reply src/test/java/seedu/address/model/AddressBookTest.java]: Didn't catch that before. Thanks! I have changed it.

**Review by luo-git — APPROVED** [review]
Great work on this PR Anthony! LGTM.

**Pull request merged**